api/osrd_infra/models/study.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.gis.db import models
+from django.contrib.postgres.fields import ArrayField
+
+from osrd_infra.schemas.study import OperationalStudyState, OperationalStudyType
+
+
+class Project(models.Model):
+    name = models.CharField(max_length=128)
+    objectives = models.CharField(max_length=255, null=True)
+    description = models.CharField(max_length=255, null=True)
+    funders = ArrayField(models.CharField(max_length=255), null=True)
+    budget = models.IntegerField(null=True)
+    image = models.ImageField(upload_to="./project_pictures", null=True)
+    studies = ArrayField(models.IntegerField(), null=True)
+    creation_date = models.DateTimeField(editable=False, auto_now_add=True)
+    last_modification = models.DateTimeField(auto_now=True)
+    tags = ArrayField(models.CharField(max_length=255), null=True)
+
+
+class OperationalStudy(models.Model):
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    name = models.CharField(max_length=128)
+    description = models.CharField(max_length=255, null=True)
+    business_code = models.CharField(max_length=128)
+    service_code = models.CharField(max_length=128)
+    creation_date = models.DateTimeField(editable=False, auto_now_add=True)
+    last_modification = models.DateTimeField(auto_now=True)
+    start_date_study = models.DateTimeField(null=True)
+    expected_end_date = models.DateTimeField(null=True)
+    actual_end_date = models.DateTimeField(null=True)
+    budget = models.IntegerField(null=True)
+    scenarios = ArrayField(models.IntegerField(), null=True)
+    tags = ArrayField(models.CharField(max_length=255), null=True)
+    state = models.CharField(max_length=16, choices=[(x.value, x.name) for x in OperationalStudyState], null=True)
+    type = models.CharField(max_length=100, choices=[(x.value, x.name) for x in OperationalStudyType], null=True)
+
+
+class Scenario(models.Model):
+    operational_study = models.ForeignKey(OperationalStudy, on_delete=models.CASCADE)
+    name = models.CharField(max_length=128)
+    description = models.CharField(max_length=255, null=True)
+    time_table_id = models.IntegerField()
+    infra_id = models.IntegerField()
+    infra_name = models.CharField(max_length=128)
```

Comment:
This info can be retrieve via the timetable. You shouldn't have duplicate information in your tables. It's going to be a nightmare to synchronize.